tests/test_cluster.go

```diff
@@ -252,6 +258,70 @@ func NewCluster(options *TestClusterConfig, logger log.Logger) (*TestCluster, er
 	return &TestCluster{testBase: testBase, archiverBase: archiverBase, host: cluster}, nil
 }
 
+func setupIndex(esConfig *esclient.Config, logger log.Logger) error {
+	esClient, err := esclient.NewIntegrationTestsClient(esConfig, logger)
```

Comment:
What's the difference between `NewIntegrationTestsClient` and `NewClient`? L180 above uses the second one.

Reply:
`NewIntegrationTestsClient` is more like admin client which supports operation that are needed for tests only to do operations which are normally made outside of server (index template creation, index creation, etc). I have TODO there to properly separate them or, vice versa, merge in one.